### PR TITLE
Check versions of installed apps

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
## Description

When we check prereqs we don't immediately check the versions of those. This is a small edition that does the check directly.

### CODE REVIEW

**Comment by tinyels** [comment]
It's worth noting that I haven't run `prereqs` in forever, have been building successfully locally and running it fails on ```WARNING: circleci not found, install via: brew install circleci```

File: scripts/prereqs [hunk 1]
```diff
@@ -47,3 +47,9 @@ else
     echo "ERROR: some prereqs missing, please install them"
     exit 1
 fi
+
+# Now check versions are correct for prereqs
+echo
+check-bash-version
+check-go-version
```
Comment (tinyels) [inline scripts/prereqs]: It's claiming that I have the wrong version of go but I thought we had already asked everyone to update to go 1.13?

Comment (chrisgilmerproj) [reply scripts/prereqs]: How far behind is your `ppp-infra` repo? The infra repo overwrites this script it appears. I will figure out how to fix that later.

**Comment by chrisgilmerproj** [comment]
> It's worth noting that I haven't run `prereqs` in forever, have been building successfully locally and running it fails on `WARNING: circleci not found, install via: brew install circleci`

`pre-commit` should only fail for you if you ever modify `.circleci/config.yml` because it would require you to have installed the `circleci` tool to validate the change. So yeah, the app will work without it but if you intend to deploy to experimental or modify any build steps you'd hit this problem.

There are certainly things you can not-install and still run/modify the app. That doesn't mean you can still modify the configuration of the app or use all the app tools without the prereqs.

File: scripts/prereqs (hunk 1 above)
Comment (chrisgilmerproj) [reply scripts/prereqs]: The go version check script has now been moved out of the ppp-infra repo :)

**Pull request merged**